.github/PULL_REQUEST_TEMPLATE

```diff
@@ -1,10 +1,32 @@
-## What changes were proposed in this pull request?
+<!--  Thanks for sending a pull request!  Here are some tips for you:
```

Comment:
How the section shows in the PR desctiption? I means `<!-- ... -->`

Reply:
It's HTML comments so it will be hidden. It will be only seen to who edits the PR description. For instance ... copy and paste this into comment and preview.

```
<!-- ... -->
```